tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
   "compilerOptions": {
-    "allowJs": false,
     "lib": ["es2015", "es2017"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "target": "es5",
+    "target": "es6",
```

Comment:
We are supporting Node.js 7.x according to the package.json / engines config. So we can compile against es6. Now application.js is 30% smaller thanks to this.

Reply:
We targeted ES5 so this would work in GCF/Lambda. Lambda has GA'd Node 8 support, but it's still [beta](https://cloud.google.com/functions/docs/concepts/nodejs-8-runtime) in GCF/Firebase. This will probably make it non functional for a couple of bots running on GCF or older versions of Lambda, so if we do this I think we should make it a breaking change and explicitly deprecate older Node versions.

Reply:
What’s the latest on this? Are we still supporting anything below Node 8?

Reply:
I feel like it's safe to remove support in Probot 8.0. All the major cloud providers where people are running Probot have at least beta support of Node 8.